h/search/core.py

```diff
@@ -168,24 +156,3 @@ def _parse_aggregation_results(self, aggregations):
         for agg in self._aggregations:
             results[agg.name] = agg.parse_result(aggregations)
         return results
-
-    @contextmanager
-    def _instrument(self):
-        if not self.stats:
-            yield
-            return
-
-        s = self.stats.pipeline()
-        timer = s.timer("search.query").start()
-        try:
-            yield
-            s.incr("search.query.success")
-        except ConnectionTimeout:
-            s.incr("search.query.timeout")
-            raise
-        except:  # noqa: E722
-            s.incr("search.query.error")
-            raise
-        finally:
-            timer.stop()
-            s.send()
```

Comment:
These search-related stats originate from https://github.com/hypothesis/h/pull/4089 Apparently we had monitoring in Grafana of how long search queries take, and of the numbers of successful, errored, and timed out search queries.